stackslib/src/burnchains/tests/mod.rs

```diff
@@ -135,14 +135,14 @@ pub struct TestMinerFactory {
 impl TestMiner {
     pub fn new(
         burnchain: &Burnchain,
-        privks: &Vec<StacksPrivateKey>,
+        privks: &[StacksPrivateKey],
```

Comment:
Since we're constructing a `Vec` from this slice, you should just pass a `Vec` (not an `&Vec`) to the function:  

```suggestion
        privks: Vec<StacksPrivateKey>,
```

This allows the caller to either give up ownership or clone, in some cases avoiding an unnecessary `Vec` construction